### PR TITLE
Revert PR #60054: add Windows.Server.Core.20H2

### PR DESCRIPTION
**Pull request by agocke**
This reverts commit efc8adbb650de1164ec886355f1ba3599e711e65.

It looks like we have a number of test failures on Server.20H2, see https://github.com/dotnet/runtime/issues/62205

Those tests will need to either be disabled or fixed before we enable this testing

### CODE REVIEW

File: eng/pipelines/libraries/helix-queues-setup.yml [hunk 1]
```diff
@@ -132,11 +132,9 @@ jobs:
             - Windows.81.Amd64.Open
             - Windows.10.Amd64.ServerRS5.Open
             - Windows.10.Amd64.Server19H1.Open
-            - Windows.10.Amd64.Server20H2.Open
```
Comment (safern) [inline eng/pipelines/libraries/helix-queues-setup.yml:135]: This one is not Server Core, are these failures happening on this SKU as well?

Comment (agocke) [reply eng/pipelines/libraries/helix-queues-setup.yml:135]: Yup, sorry, these are in 20H2 non-Core, typo

**Comment by agocke** [comment]
Confirmed in https://dev.azure.com/dnceng/public/_build/results?buildId=1493013&view=results that this passes the runtime leg

File: eng/pipelines/libraries/helix-queues-setup.yml (hunk 1 above)
Comment (agocke) [reply eng/pipelines/libraries/helix-queues-setup.yml:135]: @safern Anything else?

**Comment by agocke** [comment]
@ericstj mind signing off on this so we can unblock the build?

**Review by hoyosjs — APPROVED** [review]
Are we tracking reenabling the whole queue? The linked issue is a single test.

**Comment by agocke** [comment]
@hoyosjs Do you have a preference on re-using the existing issue on the test failures vs. opening a new issue?

**Comment by agocke** [comment]
Actually there's already an issue for this, https://github.com/dotnet/runtime/issues/57947, which was missing a 20H2 entry, which I added

**Comment by hoyosjs** [comment]
@agocke I think of them as disjoint - one tracks enabling testing on the platform, the other one tracks a flaky test on that platform. Is this the only test we've observed with this behavior?

**Comment by agocke** [comment]
I think dealing with failing tests is part of enabling the new platform. Either they need to be skipped and a bug filed, or the tests need to be fixed during bring-up. Either way, probably involves the product team.

**Comment by safern** [comment]
This will get easier once I add the new pipeline to separate what goes into rolling only and into PR 😄

**Pull request merged**